handbook/math/ceil.md

```diff
@@ -39,4 +39,4 @@ The result is `35`.
 
 ## See Also
 
-[floor](../floor) | [fix](../fix) | [round](../round)
\ No newline at end of file
+[floor](../floor) | [to-fixed](../to-fixed) | [round](../round)
```

Comment:
This is actually correct, the fix function was implemented just today: https://github.com/witheve/Eve/pull/665

The reason it's a see also here is it's closely related to floor, ceiling, and round